docs/UserGuide.md

```diff
@@ -397,17 +399,25 @@ A client can have any number of tags and insurance policies (including 0).
 
 ### <span style="color:#b573c9"><code>edit</code>: Edit client contact</span>
 
-**Purpose**: Edits an existing client contact in the ClientBook.
+**Purpose**: Edits an existing client contact in the ClientBook. The new information passed to this command will replace 
+the existing client information.
 
-**Format**: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [i/POLICY_ID[>POLICY_URL]]…​ [t/TAG]…​`
+**Format**: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [i/POLICY_ID[>POLICY_URL] [-MODE]]…​ [t/TAG]…​`
 
 * Edits the client at the specified `INDEX`.
     * `INDEX` refers to the index number shown in the displayed client list.
     * `INDEX` must be 1 or higher, and less than or equal to the index of the last item in the displayed list.
 * At least one of the optional fields must be provided.
 * Meetings of a client cannot be modified with this command.
 * If a field (e.g. `n/NAME`) is expected only once in the command, but you specified it multiple times, only the last occurrence of the field will be taken.
-    * E.g. if you specify `edit n/Alex n/Bob...`, only `n/Bob` will be taken.
+    * If you specify `edit n/Alex n/Bob...`, only `n/Bob` will be taken.
+* For fields that are allowed to be empty, specifying the identifier and leaving it empty will clear the existing information.
+    * `edit 1 i/` will clear the existing policies.
+* While editing the insurance policies of a client, user can choose to replace, modify, remove or insert policy ids to 
+a client, by typing `-MODE` to specify an editing mode after a policy number. 
+    * `-MODE` must be -insert, -modify or -remove.
+    * If user chooses to modify an existing policy, the user can specify the old and new policies, separated by `;`. An example is shown below.
+    * For each edit command, only 1 editing mode can be specified.
```

Comment:
Will be better if can include the default action (a.k.a without -MODE) which is to replace.